ui/apps/everest/src/components/nav-item/NavItem.tsx

```diff
@@ -19,6 +19,7 @@ export const NavItem = ({
   onClick,
   ...listItemProps
 }: NavItemProps) => {
+  console.log('🚀 ~ open:', open);
```

Comment:
should we then add the ignore comment here?

Reply:
never mind, it's gonna be removed, gotcha

Reply:
I left this log intentionally so that the pipeline failure would be visible